internal/infrastructure/storage/db/inmemory/market_memory_repository_impl.go

```diff
@@ -65,6 +65,22 @@ func (r MarketRepositoryImpl) GetTradableMarkets(_ context.Context) (tradableMar
 	return tradableMarkets, nil
 }
 
+//GetNonTradableMarkets returns all the markets not  available for trading
+func (r MarketRepositoryImpl) GetNonTradableMarkets(
```

Comment:
Well, here we need to get the non funded market, rather non tradable: a market could be funded but closed bacause operator decided so

Reply:
Done